### PR TITLE
Split General Config and Server Control

### PR DESCRIPTION
**Pull request by IcyPalm**
I came across this [Reddit Post](https://www.reddit.com/r/homeassistant/comments/cg6tc5/anyone_else_think_it_would_be_useful_if_there_is/) and that post mentioned exactly my bugbear while trying to set up some new automations. 

@balloob mentioned in that thread that if we knew a frontend developer to point them to the docs and let them help. Well, I do not consider myself a frontend dev(`<sarcasm>`Javascript is some form of black magic after all`</sarcasm>`) but I gave it a shot.

I still have an issue with the translations not updating but I presume I have not understood exactly how it works. Specifically with the existing strings not updating when changed like [this](https://github.com/home-assistant/home-assistant-polymer/pull/3418/files#diff-571ce32a17d6aa22fdd0942224a8ff44R586)

### CODE REVIEW

**Comment by balloob** [comment]
Hey, why did you close this ?

If you're going to write Polymer components, don't use TypeScript, as the types are madness. Just keep the files JS.

**Comment by IcyPalm** [comment]
> 
> 
> Hey, why did you close this ?
> 
I was unable to build/test the frontend with `script/build_frontend` it keeps getting stuck on `compress-static` and I did not manage to fix that locally.
`script/develop`  Did work fine though

`npm run test` fails with
`/PROJECTPATH/hassio/src/resources/hassio-icons.js
  2:28  error  Unable to resolve path to module '../../hassio-icons.html'  import/no-unresolved`

To fix that I presume I have to run `hassio/script/build_hassio` but that scripts does not seem to do anything. After 10 minutes still no output

So I am unable to run the tests locally and did not want to use GitHub as my testbed. 
This new commit is locally untested since I am not able to get the test running but it seems to work

> 
> If you're going to write Polymer components, don't use TypeScript, as the types are madness. Just keep the files JS.
>
Fixed that, seems to work now 👍

**Comment by balloob** [comment]
Just stick to `script/develop`, that's all you need. Did you see the dev docs? https://developers.home-assistant.io/docs/en/frontend_development.html

`compress-static` takes a long time, but you don't need to do prod builds to test your changes.

File: src/panels/config/server_control/ha-config-section-server-control.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        >
+          <div class="card-content">
+            [[localize('ui.panel.config.server_control.section.server_management.introduction')]]
+          </div>
+          <div class="card-actions warning">
+            <ha-call-service-button
+              class="warning"
+              hass="[[hass]]"
+              domain="homeassistant"
+              service="restart"
```
Comment (balloob) [inline src/panels/config/server_control/ha-config-section-server-control.js]: Restarting and shutting down Home Assistant should not be hidden by an advanced flag. It should be accessible to all.

File: src/panels/config/dashboard/ha-config-navigation.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ const PAGES: Array<{
   advanced?: boolean;
 }> = [
   { page: "core", core: true },
+  { page: "server_control", core: true, advanced: true },
```
Comment (balloob) [inline src/panels/config/dashboard/ha-config-navigation.ts]: Restarting and shutting down Home Assistant should not be hidden by an advanced flag. It should be accessible to all.

File: src/translations/en.json [hunk 3]
```diff
@@ -599,31 +599,37 @@
                 "imperial_example": "Fahrenheit, pounds",
                 "metric_example": "Celsius, kilograms",
                 "save_button": "Save"
-              },
-              "validation": {
```
Comment (balloob) [inline src/translations/en.json]: Before merging this, we should rename all these keys in Lokalise.

File: src/panels/config/dashboard/ha-config-navigation.ts (hunk 2 above)
Comment (IcyPalm) [reply src/panels/config/dashboard/ha-config-navigation.ts]: I have changed this. Good catch indeed. The config validation and partial reload is still hidden if advanced mode is off

**Review by balloob — APPROVED** [review]
Looks great, thanks!

**Comment by balloob** [comment]
Translation keys renamed.

**Pull request merged**